### PR TITLE
JS1: #2. Max3

### PR DESCRIPTION
**Pull request by seemcat**
Takes in 3 numbers, returns the largest.

### CODE REVIEW

File: curriculum/js1/2.js [hunk 1]
```diff
@@ -7,7 +7,9 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a > b && a > c) return a;
+  if (b > a && b > c) return b;
+  if (c > a && c > a) return c;
```
Comment (songz) [inline curriculum/js1/2.js]: you don't need to check for `c` case, because if its not a or b, it has to be c!

**Review by songz — CHANGES_REQUESTED** [review]

File: curriculum/js1/2.js (hunk 1 above)
Comment (seemcat) [reply curriculum/js1/2.js]: Resolved!

File: curriculum/js1/2.js [hunk 2]
```diff
@@ -7,7 +7,9 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a > b && a > c) return a;
+  if (b > a && b > c) return b;
+  else return c;
```
Comment (songz) [inline curriculum/js1/2.js]: no need for else.

**Review by songz — CHANGES_REQUESTED** [review]

**Review by songz — APPROVED** [review]

File: curriculum/js1/2.js (hunk 1 above)
Comment (tejupramod) [reply curriculum/js1/2.js]: Use Math.max

**Pull request merged**